src/client/pages/OfferDebugger/components/QuoteFormSweden.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { ApartmentType } from 'data/graphql'
+import { InputField } from 'components/inputs'
+import { WithFormikProps } from './QuoteData'
+
+export const initialSeApartmentValues = {
+  firstName: 'Sven',
+  lastName: 'Svensson',
+  currentInsurer: '',
+  birthDate: '1995-09-29',
+  ssn: '199509291234',
+  startDate: '',
+  email: 'sven.svensson@hedvig.com',
+  swedishApartmentData: {
```

Comment:
Here's one of the few changes in the new API: `swedishApartment` => `swedishApartmentData`

Reply:
This will use the generic api in the future but it's a little unclear how this will work at the moment so let's go with this for now.

Reply:
OK 🙃